ci/functional/test_main.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -eux
+
+test_tag=$(git show -s --format=%B | \
+           sed -ne "/^Test-tag$PRAGMA_SUFFIX:/s/^.*: *//p")
+if [ -z "$test_tag" ]; then
+    # shellcheck disable=SC2153
+    test_tag=$TEST_TAG
+fi
+
+tnodes=$(echo "$NODELIST" | cut -d ',' -f 1-"$NODE_COUNT")
+first_node=${NODELIST%%,*}
+
+clush -B -S -o '-i ci_key' -l root -w "${first_node}" \
+    "NODELIST=${NODELIST} $(cat ci/functional/setup_nfs.sh)"
+
+clush -B -S -o '-i ci_key' -l root -w "${tnodes}" \
+  "OPERATIONS_EMAIL=${OPERATIONS_EMAIL}                \
+   FIRST_NODE=${first_node}                            \
+   TEST_RPMS=${TEST_RPMS}                              \
+   $(cat ci/functional/test_main_prep_node.sh)"
+
+trap 'clush -B -S -o "-i ci_key" -l root -w "${tnodes}" '\
```

Comment:
(lint) This word is outside of quotes. Did you intend to 'nest '"'single quotes'"' instead'?  [SC2026]